src/applications/pensions/config/form.js

```diff
@@ -440,6 +441,13 @@ const formConfig = {
           uiSchema: dateOfCurrentMarriage.uiSchema,
           schema: dateOfCurrentMarriage.schema,
         },
+        currentSpouseMonthlySupport: {
+          title: 'Financial support for you spouse',
```

Comment:
you -> your

Reply:
@ToddWebDev looks like you missed this comment and it got merged with a typo :)

Reply:
@zack This spelling error won't show up in the UI. Will fix in a later commit though.